crates/wasmprinter/Cargo.toml

```diff
@@ -14,7 +14,7 @@ Rust converter from the WebAssembly binary format to the text format.
 
 [dependencies]
 anyhow = "1.0"
-wasmparser = { path = '../wasmparser', version = '0.75' }
```

Comment:
Should we drop the `version = ...` here? It is not present elsewhere.

Reply:
Anything in `[dependencies]` needs a version for publication, but `dev-dependencies` doesn't need it